include/avif/avif.h

```diff
@@ -472,6 +480,11 @@ typedef struct avifImage
     avifImageRotation irot;
     avifImageMirror imir;
 
+    // CLLI information - Content light level information. Used to represent
```

Comment:
Style nit: Please wrap all block comments in `avif.h` to 100 characters.

Please document what the defaults/absent value (presumably 0,0) should mean, and how it should be interpreted.

Reply:
Done.  I added comment to explain that (0, 0) means unknown or unavailable.